lib/player.js

```diff
@@ -7244,7 +7244,7 @@ shaka.Player.TYPICAL_BUFFERING_THRESHOLD_ = 0.5;
  * @define {string} A version number taken from git at compile time.
  * @export
  */
-shaka.Player.version = 'v4.4.0-uncompiled';
```

Comment:
FMI, if you keep it like this, will the updater replace the whole string wit the new version?

Reply:
Yes, exactly.  We lose the "-uncompiled" tag.  In compiled mode, the compiler replaces the entire symbol with a new value, so this "-uncompiled" only shows up when we load the raw source files into the browser.